deploy/docker/sysrepo-netopeer2/Dockerfile

```diff
@@ -0,0 +1,104 @@
+FROM ubuntu:18.04
+
+# defaults
+RUN \
+	apt-get update && apt-get install -y \
+	git \
+	curl \
+	wget \
+	libssh-dev \
+	libssl-dev \
+	libtool \
+	build-essential \
+	vim \
+	libtool	\
+	autoconf \
+	automake \
+	pkg-config \
+	libgtk-3-dev \
+	make \
+	vim \
+	valgrind \
+	doxygen \
+	libev-dev \
+	libpcre3-dev \
+	unzip \
+	sudo \
+	python3 \
+	build-essential \
+	bison \
+	flex \
+      swig \
+      libcmocka0 \
+      libcmocka-dev \
+      cmake
+
+# Adding netconf user
+RUN adduser --system netconf
+RUN mkdir -p /home/netconf/.ssh
+RUN echo "netconf:netconf" | chpasswd && adduser netconf sudo
+
+
+# Clearing and setting authorized ssh keys
+RUN \
+	echo '' > /home/netconf/.ssh/authorized_keys && \
+	ssh-keygen -A && \
+	ssh-keygen -t dsa -P '' -f /home/netconf/.ssh/id_dsa && \
```

Comment:
It would be best to change this to RSA keys, at least. After some fairly recent changes the allowed keys are read from the distribution config and my experience is that many of them already forbid obsolete DSA keys. Even if maybe Ubuntu 18.04 allows them, it would save us the trouble when updating to a newer release.